pkg/webhook/webhook_test.go

```diff
@@ -236,30 +236,6 @@ func TestValidRouteNoChanges(t *testing.T) {
 	expectPatches(t, resp.Patch, []jsonpatch.JsonPatchOperation{})
 }
 
-func TestInvalidOldRoute(t *testing.T) {
```

Comment:
instead of removing, can we change these tests to test that unknown fields are allowed?